src/components/Dropdown/Dropdown.md

```diff
@@ -53,7 +53,7 @@ class ControlledDropdown extends React.Component {
 
 Uncontrolled dropdown:
 
-```js { "props": { "data-description": "uncontrolled" } }
```

Comment:
Suppressed screenshot, since it there is no visual difference between it and the previous example.